src/applications/mhv-supply-reordering/containers/IntroductionPage.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { focusElement, scrollToTop } from 'platform/utilities/ui';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+// import { useSelector } from 'react-redux';
+// import { isLoggedIn } from 'platform/user/selectors';
+import { TITLE, SUBTITLE } from '../constants';
+import UnavailableSupplies from '../components/UnavailableSupplies';
+import { numAvailableSupplies, isEligible } from '../utilities/mdot';
+
+/**
+ * Introduction page for the form.
+ */
+export const IntroductionPage = props => {
+  // const userLoggedIn = useSelector(state => isLoggedIn(state));
+  const { route } = props;
+  const { formConfig, pageList } = route;
+
+  useEffect(() => {
+    scrollToTop();
+    focusElement('h1');
+  }, []);
+
+  const { mdotData } = props;
+  const numAvailSupplies = numAvailableSupplies(mdotData);
+  const canOrderSupplies = isEligible(mdotData) && numAvailSupplies > 0;
+
+  return (
+    <article className="schemaform-intro">
+      <FormTitle title={TITLE} subTitle={SUBTITLE} />
+      <p>
+        Use this form to order hearing aid batteries and accessories, and CPAP
+        supplies.
+      </p>
+      {canOrderSupplies && (
```

Comment:
Note: The intro page still needs work to add other features.

Reply:
Thanks, I was about to note that Figma file has the `<- Health care` back link